docs/specs/target.md

```diff
@@ -0,0 +1,188 @@
+  - SHOULD deploy within one Cluster or VPC at a time.
+  - Configured Vector ports, including non-default user configured ports,
+    SHOULD be automatically accessible within the Cluster or VPC.
+  - Configured Vector sources, including non-default user configured sources,
+    SHOULD be automatically discoverable via target service discovery
+    mechanisms.
+- Sizing
+  - MUST deploy in a way that takes full advantage of all system resources.
+  - The Vector service SHOULD NOT be artificially limited with resource
+    limiters such as cgroups.
+  - SHOULD request 8 vCPUs, MUST be overridable by the user.
+  - SHOULD request 2 GiB of memory per vCPU (16 GiB in this case), MUST be
+    overridable by the user.
+  - SHOULD request 6 GiB of disk space per vCPU (48 GiB in this case), MUST be
+    overridable by the user.
+  - SHOULD use the following instances if deployed in a listed cloud.
+    - AWS SHOULD default to `c6g.2xlarge` instances with 48 GiB of EBS `io2`
+      disk space.
+    - Azure SHOULD default to `f8` instances with 48 GiB of standard SSD disk
+      space.
+    - GCP SHOULD default to `c2` instances with 8 vCPUS, 16 GiB of memory, and
+      48 GiB of SSD persisted disk space.
+
+The following are additional requirements for Orchestration Platform installation
+targets:
+
+- High Availability
+  - SHOULD deploy across multiple nodes, MUST be overridable by the user.
+  - SHOULD deploy across multiple availability zones, MUST be overridable by the user.
+- Scaling
+  - MUST deploy along with a load balancer to enable horizontal autoscaling.
+  - Autoscaling SHOULD be enabled by default, driven by an average of 85%
+    CPU utilization.
+  - Autoscaling SHOULD have a stabilization period of 5 minutes.
+
+### 6. Unified Architecture
+
+The [unified architecture][unified_architecture] deploys Vector on each
+individual node for local data collection, as well as onto dedicated nodes for
+data aggregation. The requirements for both the [agent](#4-agent-architecture)
+and the [aggregator](#5-aggregator-architecture) apply to this architecture.
+This architecture SHOULD NOT be installed on Virtual/Physical Machine targets
+as there is little added benefit.
+
+## 7. Hardening
+
+- Setup
+  - An unprivileged Vector service account SHOULD be created upon installation
+    for running the Vector process.
+- Data hardening
+  - Swap SHOULD be disabled to prevent in-flight data from leaking to disk.
+    Swap would also make Vector prohibitively slow.
+  - Vector's data directory SHOULD be read and write restricted to Vector's
+    dedicated service account.
+  - Core dumps SHOULD be prevented for the Vector process to prevent in flight
+    data from leaking to disk.
+- Process hardening
+  - Vector's artifacts
+    - All communication during the setup process, such as downloading Vector
+      artifacts, MUST use encrypted channels.
+    - Downloaded Vector artifacts MUST be verified against the provided
+      checksum.
```

Comment:
Just noting we don't have these yet.

Reply:
👍 should shake out into the get targets to match spec/moving out artifact pipelines onto Datadog